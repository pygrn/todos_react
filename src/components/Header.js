import React from 'react'

const Header = () => (
  <nav>
    <div className="nav-wrapper">
      <a href="" className="brand-logo">PyGrn</a>
    </div>
  </nav>
)

export default Header