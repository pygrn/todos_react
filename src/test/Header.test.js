import React from 'react'
import Header from '../components/Header'
import renderer from 'react-test-renderer'

test('Header', () => {
    const component = renderer.create(<Header/>)
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
})