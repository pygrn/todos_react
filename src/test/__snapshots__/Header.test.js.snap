// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header 1`] = `
<nav>
  <div
    className="nav-wrapper"
  >
    <a
      className="brand-logo"
      href=""
    >
      PyGrn
    </a>
  </div>
</nav>
`;
